doc/nrf/releases_and_maturity/releases/release-notes-2.8.0.rst

```diff
@@ -0,0 +1,1550 @@
+
+Security libraries
+------------------
+
+* :ref:`nrf_security_readme` library:
+
+  * Added the :kconfig:option:`CONFIG_PSA_WANT_ALG_SP800_108_COUNTER_CMAC` Kconfig option to key derivation function configurations in :ref:`nrf_security_driver_config`.
+    The Kconfig option enables support for the derivation function SP 800-108r1 CMAC in counter mode, which is supported by the nrf_cracen driver.
+
+* :ref:`trusted_storage_readme` library:
+
+  * Added support for Zephyr Memory Storage (ZMS), as an alternative to the NVS file system.
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+* :ref:`nrf_desktop_config_channel_script` script:
+
+  * Added semantic version support for devices that use the :ref:`SUIT DFU <ug_nrf54h20_suit_dfu>`.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``a4eda30f5b0cfd0cf15512be9dcd559239dbfc91``, with some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+* Added:
+
+  * :ref:`Compressed firmware update support <mcuboot_image_compression>`.
+  * Support for SHA-512.
+  * Support for ED25519 'PURE' signature.
+    See the :kconfig:option:CONFIG_BOOT_SIGNATURE_TYPE_PURE Kconfig option in MCUboot for details.
+  * Integration of PSA crypto API as cryptographic backend for integration with cryptographic acceleration on the nRF54L15 SoC.
+    See the :kconfig:option:CONFIG_BOOT_USE_PSA_CRYPTO Kconfig option in MCUboot for details.
+  * Support for signature verification using hardware cryptography (Cracken) on the nRF54L15 SoC.
+    This is supported either for the ED25519 'PURE' signature or the ED25519 signature.
+  * Support for storing signature verification keys in hardware KMU on the nRF54L15 SoC.
+    See Kconfig option :kconfig:option:`CONFIG_BOOT_SIGNATURE_USING_KMU` in MCUboot for details.
+  * Integration of nRF54L15 SoC's HW cryptography, KMU sysbuild configuration, and signing script for building and signing the application in a project.
+    See ``SB_CONFIG_BOOT_SIGNATURE_TYPE_ED25519`` , ``SB_CONFIG_MCUBOOT_SIGNATURE_USING_KMU``, and ``SB_CONFIG_BOOT_SIGNATURE_TYPE_PURE`` Kconfig options for details.
+
+* Updated the immutable bootloader to now be protected using the :ref:`fprotect_readme` library on the nRF54L15 SoC.
+  See the ``SB_CONFIG_MCUBOOT_FPROTECT_ALLOW_COMBINED_REGIONS`` and the :kconfig:option:`CONFIG_FPROTECT_ALLOW_COMBINED_REGIONS` Kconfig option in MCUboot for details.
```

Comment:
(Related Kconfigs - ``SB_CONFIG_MCUBOOT_FPROTECT_ALLOW_COMBINED_REGIONS`` and :kconfig:option:`CONFIG_FPROTECT_ALLOW_COMBINED_REGIONS`)